.github/workflows/main.yml

```diff
@@ -198,23 +198,22 @@ jobs:
         include:
           - name: ubu24
             os: ubuntu-24.04-arm
```

Comment:
Change to ubuntu-24.04 
for now. I see some error involving mamba here. I've raised it to the mamba folks !